### PR TITLE
VLT-1914: add apple pay for nas specs

### PR DESCRIPTION
**Pull request by rich-cooper-cko**
Nas Preview Link -> https://checkout-api-reference-git-task-vlt-1914-apple-805296-cko-docs.vercel.app/preview/crusoe/

### CODE REVIEW

File: nas_spec/components/schemas/ApplePay/ApplePaySigningRequestResponse.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+type: object
+description: Retrieve a signing request for a client to begin certification process
+required:
+  - value
+properties:
+  content:
+    type: string
+    description: The signing request content
+    example: -----BEGIN CERTIFICATE-----MIIBSDCB8AIBADCBjzELMAkGA1UEBhMCR0IxDzANBgNVBAgMBkxvbmRvbjEPMA0GA1UEBwwGTG9uZG9uMRUwEwYDVQQKDAxDaGVja291dC5jb20xCzAJBgNVBA8MAklUMRUwEwYDVQQDDAxjaGVja291dC5jb20xIzAhBgkqhkiG9w0BCQEWFHN1cHBvcnRAY2hlY2tvdXQuY29tMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEXf/MDFRLSblykajGm0GE+lUNoOIEa0DbWc7Bv3s55bYtW1fJo2/MJIPojUKuKUx2VsEfGmmqXKbq4IhAr0bM8TAKBggqhkjOPQQDAgNHADBEAiAo1Dv4TXTacSeIfy4HDjzPMQY2+OxTW6szRJjGyfKgXQIgdHAX0BmI+1rozysjXv8MvoxehQIGACQ+UWJle+UZ2ms=-----END CERTIFICATE-----
```
Comment (ashley-barrett-cko) [inline nas_spec/components/schemas/ApplePay/ApplePaySigningRequestResponse.yaml:9]: BEGIN NEW CERTIFICATE REQUEST

Comment (ashley-barrett-cko) [reply nas_spec/components/schemas/ApplePay/ApplePaySigningRequestResponse.yaml:9]: END NEW CERTIFICATE REQUEST

**Review by ashley-barrett-cko — CHANGES_REQUESTED** [review]

File: nas_spec/components/schemas/ApplePay/ApplePaySigningRequestResponse.yaml (hunk 1 above)
Comment (rich-cooper-cko) [reply nas_spec/components/schemas/ApplePay/ApplePaySigningRequestResponse.yaml]: I checked the api and it came back as `BEGIN CERTIFICATE REQUEST`. Updated accordingly

File: nas_spec/swagger.yaml [hunk 2]
```diff
@@ -63,6 +63,8 @@ tags:
     description: Use Workflows to start receiving webhook notifications for your payments, disputes, and marketplace sub-entities.
   - name: Risk
     description: Evaluate the risk of a transaction at the pre-authentication and pre-capture stages of the payment process.
+  - name: Apple Pay
+    description: Manage your apple pay certificates in order to process via apple pay
```
Comment (toby-knott-cko) [inline nas_spec/swagger.yaml:67]:
```suggestion
    description: Manage your Apple Pay certificates in order to process payments via Apple Pay.  
```

File: nas_spec/swagger.yaml [hunk 3]
```diff
@@ -91,3 +93,6 @@ x-tagGroups:
   - name: Marketplace
     tags:
       - Marketplace
+  - name: Apple Pay
+    tags:
+      - Apple Pay
```
Comment (toby-knott-cko) [inline nas_spec/swagger.yaml:98]: I think it makes sense to have these Apple Pay endpoints part of the 'Handle payments and payouts section' (underneath Forex). What do you think?

**Review by toby-knott-cko — CHANGES_REQUESTED** [review]

File: nas_spec/swagger.yaml (hunk 3 above)
Comment (rich-cooper-cko) [reply nas_spec/swagger.yaml:98]: Sure I can move these

**Review by toby-knott-cko — DISMISSED** [review]

**Review by anthony-simpson-cko — DISMISSED** [review]

**Review by ashley-barrett-cko — APPROVED** [review]

**Review by toby-knott-cko — APPROVED** [review]

**Pull request merged**